docs/faq.md

```diff
@@ -152,6 +152,23 @@ In either case, look at the
 [coverage reports]({{ site.baseurl }}/further-reading/clusterfuzz#coverage-reports)
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## What if my fuzzer does not find new coverage or bugs after a while?
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
```

Comment:
the formatting of this seems arbitrary to me. Maybe try to do one sentence per line or an 80-char limit.